### PR TITLE
Fixing CVE-2020-24455 in `tpm2-tss`

### PR DESCRIPTION
**Pull request by PawelWMS**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Updating `tpm2-tss` to version 2.4.6 to fix CVE-2020-24455.

###### Change Log  

- Updated `tpm2-tss` to version 2.4.6.
- Updated spec to use `make` build and install macros, and `%autosetup`.

###### Does this affect the toolchain?  

No.

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2020-24455

###### Test Methodology

- Local package build.

### CODE REVIEW

**Comment by k-a-suvorov** [comment]
Спасибо, Ваше письмо получено.
С уважением, Константин Суворов

File: SPECS/tpm2-tss/tpm2-tss.spec [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
     --disable-doxygen-doc \
-    --with-udevrulesdir=/etc/udev/rules.d
+    --with-udevrulesdir=%{_sysconfdir}/udev/rules.d
 
-make %{?_smp_mflags}
+%make_build
 
 %install
-make DESTDIR=%{buildroot} install
+%make_install
```
Comment (anphel31) [inline SPECS/tpm2-tss/tpm2-tss.spec:45]: do you think we can enable the check tests? 
https://github.com/tpm2-software/tpm2-tss#testing

**Review by anphel31 — APPROVED** [review]

File: SPECS/tpm2-tss/tpm2-tss.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/tpm2-tss/tpm2-tss.spec:45]: looks like the unit tests require cmocka which is not available yet. However, I see this being added in #1180

Comment (PawelWMS) [reply SPECS/tpm2-tss/tpm2-tss.spec:45]: I doubt I can get integration tests to work due to the dependency on `uthash`. I'll try to enable unit tests once we get a build with `cmocka` and its dependencies.

**Pull request merged**